src/both/DeltaNeutralLpV3.sol

```diff
@@ -131,6 +144,36 @@ contract DeltaNeutralLpV3 is AccessStrategy {
     /// @dev Aave deposit receipt token.
     ERC20 public immutable aToken;
 
+    /*//////////////////////////////////////////////////////////////
+                          PRICE CONVERSION
+    //////////////////////////////////////////////////////////////*/
+
+    /**
+     * @notice abs(asset.decimals() - borrow.decimals() - borrowFeed.decimals()). This value has significance
+     * while converting asset amount to borrow amount and vice versa.
```

Comment:
The second sentence can say `Used when converting between asset/borrow amounts`. You can probably make this a single line comment and use `///`.